app/src/main/res/values-uk/strings.xml

```diff
@@ -5,10 +5,10 @@
     <string name="dark">Темна</string>
     <string name="delete">Видалити</string>
     <string name="rename">Перейменувати</string>
-    <string name="confirm_delete">Підтвердити видалення</string>
+    <string name="confirm_delete">Підтвердіть видалення</string>
```

Comment:
I am not sure that this makes sense. If it is button title than it should be reverted. Because previous means action and new one - request. @gsantner When this one is used?

Reply:
When you select an item, click on the delete button. then you asked if you REALLY want to delete that file(s)

Reply:
It is appeared on BUTTON or dialog title / content?

Reply:
it's in a dialog's title https://github.com/gsantner/markor/blob/d39f80116cad7daf209087a6f033f2494d0a256d/app/src/main/java/net/gsantner/markor/activity/FilesystemListFragment.java#L146

Reply:
@cl0ne super. If so you, I am sorry.

Reply:
@Abrasha np